### PR TITLE
Ml core 4.3

### PR DESCRIPTION
**Pull request by priyanka-TL**

### CODE REVIEW

**Comment by amorphous-1** [comment]
Hello @priyanka-TL,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://862.qa.docs.sunbird.org/4.3.0/ .<br/> **Note:** Only _4.3.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://862.qa.docs.sunbird.org/4.3.0/apis/ml-core// <br /> - http://862.qa.docs.sunbird.org/4.3.0/apis/ml-core// <br /> - http://862.qa.docs.sunbird.org/4.3.0/apis/ml-projects// <br /> - http://862.qa.docs.sunbird.org/4.3.0/apis/ml-projects// <br /> - http://862.qa.docs.sunbird.org/4.3.0/apis/ml-reports// <br /> - http://862.qa.docs.sunbird.org/4.3.0/apis/ml-reports// <br /> - http://862.qa.docs.sunbird.org/4.3.0/apis/ml-survey// <br /> - http://862.qa.docs.sunbird.org/4.3.0/apis/ml-survey// <br />  <br/> **Modified sections are -** <br />

**Comment by amorphous-1** [comment]
Hello @priyanka-TL,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://862.qa.docs.sunbird.org/4.3.0/ .<br/> **Note:** Only _4.3.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://862.qa.docs.sunbird.org/4.3.0/apis/ml-core// <br /> - http://862.qa.docs.sunbird.org/4.3.0/apis/ml-core// <br />  <br/> **Modified sections are -** <br />

File: apis/ml-core/ml-coreapi.yaml [hunk 1]
```diff
@@ -0,0 +1,1290 @@
...
+          description: >-
+            To make use of the API, you require authorization. Raise a request
+            to the administrator for the use of the API. You will receive the
+            authorization key. Specify the key received, here.
+          required: true
+          schema:
+            type: string
+        - name: page
+          in: query
+          description: Page Number
```
Comment (mohitga) [inline apis/ml-core/ml-coreapi.yaml:47]: please add a description to the query parameter, Are these parameters used as filters?

File: apis/ml-core/ml-coreapi.yaml [hunk 2]
```diff
@@ -0,0 +1,1290 @@
...
+          schema:
+            type: string
+          example: 2
+        - name: limit
+          in: query
+          description: Number of objects to be returned
```
Comment (mohitga) [inline apis/ml-core/ml-coreapi.yaml:53]: Are these the total number of records returned or the number of records returned per page?

File: apis/ml-core/ml-coreapi.yaml [hunk 3]
```diff
@@ -0,0 +1,1290 @@
...
+          application/json:
+            schema:
+              $ref: "#/components/schemas/userapiUserProgramAndSolutionRequest"
+      responses:
+        200:
+          description: "Program Response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/userapiProgramResponse"
```
Comment (mohitga) [inline apis/ml-core/ml-coreapi.yaml:82]: Also add 400 error response

File: apis/ml-core/ml-coreapi.yaml [hunk 4]
```diff
@@ -0,0 +1,1290 @@
...
+                $ref: '#/components/schemas/500Error'
+      deprecated: false
+      security:
+        - bearer: []
+  /users/mlcore/v1/solutions{programId}:
+    post:
+      summary: User targeted solutions.
+      description: |-
+        This API is used to get list of targeted user solutions
+          - The endpoint for **User Programs** is `/users/mlcore/v1/solutions`
```
Comment (mohitga) [inline apis/ml-core/ml-coreapi.yaml:97]: replace with {/users/mlcore/v1/solutions{programId}}

File: apis/ml-core/ml-coreapi.yaml [hunk 5]
```diff
@@ -0,0 +1,1290 @@
...
+           - JSON 
+           
+          The body is the representation of the resource object for searching objects
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/userapiUserProgramAndSolutionRequest"
+      responses:
+        200:
+          description: "Program Response"
```
Comment (mohitga) [inline apis/ml-core/ml-coreapi.yaml:78]: Should be, list of program return based on the parameter provided

File: apis/ml-core/ml-coreapi.yaml [hunk 6]
```diff
@@ -0,0 +1,1290 @@
...
+          required: true
+          schema:
+            type: string
+        - name: Authorization
+          in: header
+          description: >-
+            To make use of the API, you require authorization. Raise a request
+            to the administrator for the use of the API. You will receive the
+            authorization key. Specify the key received, here.
+          required: true
```
Comment (mohitga) [inline apis/ml-core/ml-coreapi.yaml:42]: internal-access-token is missing

File: apis/ml-core/ml-coreapi.yaml [hunk 7]
```diff
@@ -0,0 +1,1290 @@
...
+          schema:
```
Comment (mohitga) [inline apis/ml-core/ml-coreapi.yaml:43]: Add query parameter in the example

**Pull request merged**